### PR TITLE
removing soft fail for 13.2 because bug #949188 is fixed

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

File: tests/installation/setup_zdup.pm [hunk 1]
```diff
@@ -49,11 +49,6 @@ sub run() {
         select_console('root-console');
     }
 
-    if (get_var('HDD_1', '') =~ /opensuse-13\.2/) {
-        record_soft_failure 'bsc#949188, kernel panic on 13.2';
-        assert_script_run("zypper -n rm apparmor-abstractions");
-    }
 }
-
```
Comment (okurz) [inline tests/installation/setup_zdup.pm]: please keep the separating blank line. We commonly have this in our files.

**Comment by asmorodskyi** [comment]
@okurz , @lnussel please review this PR .

**Comment by okurz** [comment]
Our convention is that the git commit subject lines should be stated in imperative mode and start with capital letters. Also see http://chris.beams.io/posts/git-commit/ as a helpful guide how to write good commit messages.

**Comment by asmorodskyi** [comment]
1. revert deletion of line 
2. change commit messages according to link provided by @okurz

**Comment by lnussel** [comment]
+1 for the change in general but the test fail...

File: tests/installation/setup_zdup.pm [hunk 2]
```diff
@@ -8,8 +8,8 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# G-Summary: Rework the tests layout.
-# G-Maintainer: Alberto Planas <aplanas@suse.com>
+# Summary: Preparation step for zypper dup. Making sure that a console is available and selected. 
+# Maintainer: Stephan Kulow <coolo@suse.com>
```
Comment (okurz) [inline tests/installation/setup_zdup.pm]: `tools/tidy` complains that you put w/s at EOL.

**Comment by okurz** [comment]
> I'm not the maintainer of this. I didn't even touch it beside minor cleanup

that's why asmorodskyi asked you in #os-fctry if you are ok to take maintainership. I guess it could be @lnussel then?

**Comment by asmorodskyi** [comment]
1. @coolo  I replace you as maintainer with @lnussel 
2. remove whitespices

**Pull request merged**